packages/feedback/src/styles.scss

```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
If you are not using this file please delete this

Reply:
this file is needed.if i am removing this file listing page functionality is getting break.